server.js

```diff
@@ -17,9 +18,11 @@ var path = require('path'),
     password = process.env.PASSWORD,
     env      = process.env.NODE_ENV || 'development',
     useAuth  = process.env.USE_AUTH || config.useAuth;
+    useAutoStoreData = config.useAutoStoreData;
 
     env      = env.toLowerCase();
     useAuth  = useAuth.toLowerCase();
+    useAutoStoreData = useAutoStoreData.toLowerCase();
```

Comment:
I think we could go back and do similar for `useAuth`. Boolean feels less fragile to me.